packages/ui/src/documents/__tests__/EditDocument.test.tsx

```diff
@@ -114,57 +122,12 @@ describe('Edit Document', () => {
       expect(documentForm.find(Nfts).prop('viewMode')).toBe(false);
       expect(documentForm.find(FundingAgreements).prop('viewMode')).toBe(false);
     });
-
-  });
-
-  it('Should not render funding agreements', async () => {
```

Comment:
Same as above (test coverage deterioration)

Reply:
we don't use funding agreements at all in the current Gateway setup